examples/src/docker/main.rs

```diff
@@ -67,12 +72,44 @@ async fn run(args: Args, token: CancellationToken) -> Result<()> {
                 .image("alpine")
                 .program("echo")
                 .args([String::from("hello, world!")])
+                .stdout("/stdout")
```

Comment:
Should these also be added for the TES example? Or am I missing something?

Reply:
Yep, good catch (I've confirmed the Docker sample, but not the TES one).